src/Microsoft.Identity.Client/Internal/MsalHelpers.cs

```diff
@@ -35,6 +35,16 @@ namespace Microsoft.Identity.Client.Internal
 {
     internal static class MsalHelpers
     {
+        public static string ToStringInvariant(this int val)
```

Comment:
i am using the exact naming that .NET uses and it is ToLowerInvariant().

---
In reply to: [114103738](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/378#discussion_r114103738) [](ancestors = 114103738)